boards/stm32f0discovery/dist/debug.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/sh
 
-echo "Debugging $1"
-arm-none-eabi-gdb -tui -command=$1 $2
+openocd -f "board/stm32f0discovery.cfg" \
+    -c "tcl_port 6333" \
+    -c "telnet_port 4444" \
+    -c "init" \
```

Comment:
I don't really see the reason for this, normally you configure your gdb client (i.e. Eclipse) for a certain server port and not your server to the port your client wants...